### ISSUE TITLE
Improve Kiali wizards UI

### ISSUE THREAD

**Issue by ferhoyos** [kiali/kiali#7451]
### What do you want to improve?

I want to improve the Kiali wizards to be more consistent across the Kiali application and to follow the best UX practices.

### What is the current behavior?

Currently, there are two types of Kiali wizards:
- **Creation of new Istio objects:** This wizard is a form page where each section is displayed vertically throughout the wizard.

![image](https://github.com/kiali/kiali/assets/122779323/ed958960-b31c-484e-ae76-9b12dfb14273)

- **Service routing:** This wizard is a modal where each section is displayed with tabs.

![image](https://github.com/kiali/kiali/assets/122779323/ecd26cef-388d-43b7-8a1d-9aa82798109c)

IMO Kiali should follow the same UX approach for all the wizards to provide consistency across the application.

**Which approach is better?**

The creation of new Istio object wizard is the approach to follow for these reasons:
- All wizards end with a preview modal that displays the YAML to be generated. The service wizard is a modal, and opening a modal from another modal is not a good practice in UX.
- Tab sections in a wizard can be confusing. The user may not know if each tab is an independent section or if they are correlated in some way.

### What is the new behavior?

Convert service wizards into form pages and display each section vertically throughout the wizard, similar to the new Istio object creation wizards.

**Comment by jshaughn** [comment]
@ferhoyos I'm adding this to the sprint backlog, but as a research effort with Foday, to come up with a design that we should apply.  Try to balance effort and usability, if it is too much work it likely won't get done.

**Comment by ferhoyos** [comment]
I can list multiple efforts we can take to improve Kiali UI wizards (e.g., converting modals into forms, removing tabs, improving type validations, etc.). I think the best approach here is to convert this issue into an epic to allow a smooth transition with multiple issues/PRs. Additionally, we can estimate each task separately and decide if it is worth based on the effort/usability balance.  Is this feasible from a UX perpsective @fkargbo?

**Comment by jshaughn** [comment]
@ferhoyos I'm moving this Epic to in-progress, and co-assigning to you and [@mayleighnmeyers.](https://github.com/mayleighnmyers)  When you two decide it's a good time you can transfer sole ownership to Mayleigh.  I added this as a topic to the UX meeting tomorrow, please take it off the agenda if you think it's too early. Thanks!

**Comment by jshaughn** [comment]
@ferhoyos As discussed, let's limit the current work on this Epic to near-term tasks. Near-term would be anything that improves wizard consistency but doesn't significantly change the forms layout, or take significant investment (minimal UX). Longer term items can be left fairly vague, as this Epic will likely be moved out of the sprint backlog when the near-term tasks are completed.  Thanks.

**Comment by jshaughn** [comment]
Moving back to Backlog until we have resources that can work on it.

**Comment by jmazzitelli** [comment]
I think we should seriously consider removing the wizards entirely. I'm guessing hardly any user/customer uses them, it's a maintenance burden, and well... there. I said it. :)

**Comment by jshaughn** [comment]
I don't agree that we should remove the wizards, they can be quite useful when needed. But I do agree that they get limited use and are likely not used in production scenarios. As such, this effort currently has lower priority.

**Comment by jshaughn** [comment]
Community Note: Any changes here need UX/Design approval before coding/PR submission.
